tests/helix/send-to-helix-inner.proj

```diff
@@ -75,7 +75,7 @@
 
     <HelixPreCommand Condition="'$(OS)' != 'Windows_NT'" Include="chmod +x $HELIX_WORKITEM_ROOT/.playwright/node/linux-x64/node" />
     <!-- wait for lock for 2mins -->
-    <HelixPreCommand Condition="'$(OS)' != 'Windows_NT'" Include="sudo apt-get -o DPkg::Lock::Timeout=120 install -y libgbm1 libxkbcommon0 || exit 1" />
+    <HelixPreCommand Condition="'$(OS)' != 'Windows_NT'" Include="(sudo apt-get -o DPkg::Lock::Timeout=120 install -y libgbm1 libxkbcommon0 || (cat /var/lib/dkms/lttng-modules/2.13.8/build/make.log; exit 1)) || exit 1" />
```

Comment:
> /var/lib/dkms/lttng-modules/2.13.8

What happens when a new version is shipped?

Reply:
This is very specifically to debug a helix issue being hit right now. There's no other reason to dump such a specific file otherwise. So, I will remove this once we have the needed information.